src/components/landingPage/__snapshots__/LandingPage.test.js.snap

```diff
@@ -2,28 +2,38 @@
 
 exports[`LandingPage component renders App 1`] = `
 <div
-  className="LandingPage"
+  className="Full"
 >
-  <h1>
-    Choose Nickname:
+  <h1
+    className="Logo"
+  >
+    Happy Trees
   </h1>
   <div
-    className="Form"
+    className="LandingPage"
   >
-    <form
-      onSubmit={[Function]}
+    <h1>
+      Choose Nickname:
+    </h1>
+    <div
+      className="Form"
```

Comment:
Looks like you added a div around the form for styling. Might be better to name the class FomContainer instead of just Form. Otherwise looks good.